modules/go-health-monitoring.adoc

```diff
@@ -0,0 +1,19 @@
+[id="health-information-resources_{context}"]
+= Checking health information
+
+.Prerequisites
+
+* {gitops-title} Operator installed from *OperatorHub*.
```

Comment:
I would add the article "The" in the beginning, as well as rewrite it to "is installed"